docs/designers-developers/developers/data/data-core-editor.md

```diff
@@ -294,6 +294,31 @@ _Returns_
 
 -   `string`: Post type.
 
+<a name="getDefaultTemplateType" href="#getDefaultTemplateType">#</a> **getDefaultTemplateType**
```

Comment:
Should all three new API methods be marked unstable or even experimental to give them some time to mature? This way they won't be exposed in docs until we are 100% sure that we can support them indefinitely 😃

@adamziel has recently extracted store handling related to reusable blocks to its own store and moved to an isolated package. It's something that should be discussed in this context as well. I'd like to get some feedback from @youknowriad who has the best overview how editor packages are architected.

Reply:
Yes, you are totally right.
These selectors originated in `edit-site` where we work under the assumption that everything is experimental, and didn't think of explicitly marking them as such when moving them here.
I'll definitely mark them as `__experimental` if we decide that this approach is the right one!

Reply:
Eh, I went ahead and done it anyway. 😄 

TIL about the `reusable-blocks` package!
It sounds like a very neat idea, that in our case could be used to store all FSE-related functions that might be used in different editors and contexts.

As it always is, naming is hard!
`templates` sounds clean and nifty, but FSE is not just templates (there are also... template _parts_! 😄 )
`full-site-editing` would be catch-all, but it sounds slightly awkward to me.

Also, I've never created a new package.
Is there anything I should know to make it work smoothly?
How would publishing it to npm work?

Reply:
This should help with a new package:
https://github.com/WordPress/gutenberg/tree/master/packages#creating-a-new-package

You can wait a bit more with that step. It might help to better understand what scope such package would have. We have also [`@wordpress/media-utils`](https://github.com/WordPress/gutenberg/tree/master/packages/media-utils) that is catch it all for media related code 😄  

Publishing is automated, so you don't have to worry about it other than set some version lower than 1.0 to make it clear that it's all experimental.

I'm checking https://github.com/WordPress/gutenberg/pull/25859 to learn if there is anything that isn't included in the docs but I think everything else will catch CI. I assume you will have to run `npm run docs:build` to expose README in the documentation portal at https://developer.wordpress.org/block-editor/packages/. It should add a section like this in the manifest file:
https://github.com/WordPress/gutenberg/blob/3aeb89e6fdb8a72741ea9078f199f8265ffb8592/docs/manifest.json#L1670-L1675

Reply:
Thanks @gziolo !
I've started working on it in https://github.com/WordPress/gutenberg/pull/27148 (not ready and not working btw 😄) mostly to learn more about how packages work and interact between each other.

I think right now there isn't that much stuff to justify a new package, but maybe in the future things will change. 👍